sparql/sparql12/rdf11/manifest.ttl

```diff
@@ -0,0 +1,37 @@
+@prefix rdf:    <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix :       <https://w3c.github.io/rdf-tests/sparql/sparql12/grouping#> .
+@prefix rdfs:   <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix mf:     <http://www.w3.org/2001/sw/DataAccess/tests/test-manifest#> .
+@prefix qt:     <http://www.w3.org/2001/sw/DataAccess/tests/test-query#> .
+@prefix dawgt:  <http://www.w3.org/2001/sw/DataAccess/tests/test-dawg#> .
+
+:manifest  rdf:type mf:Manifest ;
+    rdfs:label "RDF 1.1 literals handling" ;
+    mf:entries
+    (
+    :langstring-datatype
+    :plain-string-datatype
+    :plain-string-same
+    ) .
+
+
+:langstring-datatype rdf:type mf:QueryEvaluationTest ;
+    mf:name "DATATYPE on rdf:langString";
+    rdfs:comment    "plain literals with a language tag have a rdf:langString datatype" ;
+    dawgt:approval dawgt:Proposed ;
+    mf:action [ qt:query  <langstring-datatype.rq> ] ;
+    mf:result  <langstring-datatype.srj> .
+
+:plain-string-datatype rdf:type mf:QueryEvaluationTest ;
+    mf:name "DATATYPE on plain string literal";
+    rdfs:comment    "plain literals without a language tag have the xsd:string datatype" ;
```

Comment:
It seems like these `rdfs:comment` should be phrased alike, either `have a ... datatype` or `have the ... datatype`. I'm inclined toward the latter.